src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+@import './body';
+@import './grade';
+@import './price';
+
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__items {
+  object-fit: contain;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  margin: 0 16px 4px;
+  color: $main-text-color;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+  margin: 0 0 16px;
+  padding-left: 16px;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  background-color: #00acdc;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  margin: 0 16px;
+}
```

Comment:
The `.button` class has a fixed width and height, which may not be flexible for different text lengths or localization. Consider using padding instead of fixed width and height to allow the button size to adjust based on its content.